joplin/base/models/department_page.py

```diff
@@ -72,13 +75,17 @@ def __str__(self):
     base_form_class = DepartmentPageForm
 
     content_panels = [
-        FieldPanel('title_en'),
-        FieldPanel('title_es'),
+        FieldPanel('title_en', widget=countMe),
+        FieldPanel('title_es', widget=countMe),
         FieldPanel('title_ar'),
```

Comment:
while it would be nice to avoid needing to put all of these titles by language in here (I'm not sure if we have an issue on the board for cleaning this up), it'd be good to update the widget on all of these.

Also this raises the question: do we have different character limits for different languages?

Reply:
We do not, but with this widget we can now make that more possible.